src/main/java/core/mate/academy/service/TruckProducer.java

```diff
@@ -0,0 +1,29 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Truck;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TruckProducer implements MachineProducer<Truck> {
+    private List<Truck> trucks = new ArrayList<>();
```

Comment:
Don't forget about access modifiers for your custom fields. The 'trucks' field should be private to follow the encapsulation principle.